README.md

```diff
@@ -11,3 +11,292 @@
+            published: false, // Whether the post has been released to the general public
+            body: '',         // Body text of this post
+            logo: null,       // The logo to show for this post
+            author: 'John Doe'// Who wrote the post?
+        }
+    });
+});
+```
+
+Here is an example of how the defaults get used:
+
+```javascript
+var post = model('posts').create({});
+console.log(post.author);
+=> John Doe
+```
+
+## Creating instances of your model
+
+You can use angular-model ad-hoc to construct object instances:
+
+```javascript
+// From defaults
+var post = model('posts').create({});
+
+// Specifying fields
+var post = model('posts').create({
+  name: 'some post',
+  body: "body of some body, it's just some body, you know?",
+  author: 'Steve Davis'
+});
+
+console.log(post.author);
+=> Steve Davis
+```
+
+## Instance Methods
+
+angular-model instances have instance methods, similar to objects in the OOP world.
+
+### Default instance methods
+
+The following methods are available to every angular-model instance.
+
+Function | Description
+------- | -----------
+$save | Persist an instance to the API
+$delete | Tell the API to delete an instance
+$reload | Refresh an instance of a model from the API
+$revert | Reset the model to the state it was originally in when you first got it from the API
+$exists | Reset the model to the state it was originally in when you first got it from the API
+$dirty | Returns boolean - true if a model instance has been modified, else false. Opposite of $pristine.
+$pristine | Returns boolean - true if a model instance has unmodified, else false. Opposite of $pristine.
+$related | Hydrates the $links property of the instance. $links are used so that an instance can tell the client which objects are related to it. For example, a `post` may have an `author` object related to it.
+$modified | Returns a map of the properties that have been changed
+$hasRelated | Does an instance have a relation of name `name`?
+
+> You can see full details of these methods in the [API documentation](/docs/api.md).
+
+### Custom instance methods
+
+angular-model allows you to define instance methods on instances. This is similar to adding methods by extending a base class in the OOP world.
+
+```javascript
+yourApp.config(function(modelProvider) {
+    modelProvider.model('posts', {
+        // ...
+
+        /**
+         * @ngdoc object
+         * @name yourApp.posts.$instance
+         * @description
+         * Instance methods that are callable on any individual instance of a post
+         */
+        $instance: {
+            /**
+             * @ngdoc function
+             * @name yourApp.posts.$logo
+             * @description
+             * If this post instance has a logo, return it, otherwise return a default string
+             *
+             * @return string Either the logo for this post, or a default logo
+             */
+            $logo: function() {
```

Comment:
Actually, ignore me. I was gonna say rename it to `getLogo()` or something, but methods probably _should_ always be prefixed to ensure they never conflict with API key names.